src/views/Onboarding/OnboardingPassword.vue

```diff
@@ -93,6 +93,13 @@ export default {
     margin-top: 2rem;
   }
 
+  &_errorLength {
+    background: #fff;
+    border-top: 1.75px solid #F12274;
```

Comment:
`1.75px` should be either 1 or 2
any color used should be a sass variable. for errors there is `$danger`